README.md

```diff
@@ -71,6 +71,22 @@ Press CTRL+C to quit
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+## Deploying the App
+
+Deployment is managed with Ansible.
+
+Create an [inventory file](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html)
+with a `webservers` group containing all the hosts to deploy the app to.
```

Comment:
it felt weird to commit the inventory file as it contained the IP of the host to deploy to

what's standard here?

Reply:
For this exercise, just committing it is standard (if nothing else, it makes my life a bit easier remembering which host to check! 😛 ). 
For a real project, you're right, I wouldn't want the server IPs in a public repo certainly (I don't think I'd be _very_ fussed in a private one, but might depend on sensitivity). The answer to what's standard likely depends on what happens next though, e.g.:
* If your servers are being created by Terraform, then potentially just generate the hosts file programmatically using Terraform output
* If you have a pipeline that runs your Ansible against fixed addresses, maybe you inject them as pipeline secrets somehow
* If you're running them manually on a controller, maybe you just leave the file on the controller (and record it in your docs somewhere)
etc.